tests/utils/tests/test_mimebundle.py

```diff
@@ -40,170 +40,6 @@ def vegalite_spec():
-                            "signal": '"a: " + (isValid(datum["a"]) ? datum["a"] : ""+datum["a"]) + "; b: " + (format(datum["b"], ""))'
-                        },
-                        "fill": {"value": "#4c78a8"},
-                        "width": {"band": 1, "scale": "x"},
-                        "x": {"field": "a", "scale": "x"},
-                        "y": {"field": "b_end", "scale": "y"},
-                        "y2": {"field": "b_start", "scale": "y"},
-                    }
-                },
-                "from": {"data": "data_0"},
-                "name": "marks",
-                "style": ["bar"],
-                "type": "rect",
-            }
-        ],
-        "padding": 5,
-        "scales": [
-            {
-                "domain": {"data": "data_0", "field": "a", "sort": True},
-                "name": "x",
-                "paddingInner": 0.1,
-                "paddingOuter": 0.05,
-                "range": {"step": {"signal": "x_step"}},
-                "type": "band",
-            },
-            {
-                "domain": {"data": "data_0", "fields": ["b_start", "b_end"]},
-                "name": "y",
-                "nice": True,
-                "range": [{"signal": "height"}, 0],
-                "type": "linear",
-                "zero": True,
-            },
-        ],
-        "signals": [
-            {"name": "x_step", "value": 20},
-            {
-                "name": "width",
-                "update": "bandspace(domain('x').length, 0.1, 0.05) * x_step",
-            },
-        ],
-        "style": "cell",
-    }
-
-
-@pytest.mark.save_engine
-@pytest.mark.parametrize("engine", ["vl-convert", "altair_saver", None])
-def test_vegalite_to_vega_mimebundle(engine, vegalite_spec, vega_spec):
-    if engine == "vl-convert" and vlc is None:
-        pytest.skip("vl_convert not importable; cannot run mimebundle tests")
-    elif engine == "altair_saver" and altair_saver is None:
-        pytest.skip("altair_saver not importable; cannot run mimebundle tests")
-    elif vlc is None and altair_saver is None:
-        pytest.skip(
-            "Neither altair_saver nor vl_convert are importable;"
-            + " cannot run mimebundle tests"
-        )
-
-    # temporary fix for https://github.com/vega/vega-lite/issues/7776
-    def delete_none(axes):
-        for axis in axes:
-            for key, value in list(axis.items()):
-                if value is None:
-                    del axis[key]
-        return axes
-
-    bundle = spec_to_mimebundle(
-        spec=vegalite_spec,
-        format="vega",
-        mode="vega-lite",
-        vega_version=alt.VEGA_VERSION,
-        vegalite_version=alt.VEGALITE_VERSION,
-        vegaembed_version=alt.VEGAEMBED_VERSION,
-        engine=engine,
-    )
-
-    bundle["application/vnd.vega.v5+json"]["axes"] = delete_none(
-        bundle["application/vnd.vega.v5+json"]["axes"]
-    )
-    assert bundle == {"application/vnd.vega.v5+json": vega_spec}
-
-
```

Comment:
It looks like this vl-convert test was accidentally deleted

Reply:
Nevermind, I can see that this is only for conversion to the Vega mimebundle now, so it should be deleted